src/test/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMergerCoversheetsTest.java

```diff
@@ -0,0 +1,235 @@
+package uk.gov.hmcts.reform.em.stitching.pdf;
```

Comment:
This file isn't the cleanest. Let me know if you want me to split these assertions into smaller tests, and if you want me to work on getting this code more DRY